media.emby.EmbyServer.json

```diff
@@ -0,0 +1,29 @@
+{
+    "app-id": "media.emby.EmbyServer",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "media.emby.EmbyServer",
+    "finish-args": ["--share=network", "--filesystem=host", "--device=dri"],
+    "modules": [
+        {
+            "name": "media.emby.EmbyServer",
```

Comment:
Nothing wrong with the manifest per se, but: 1) is it actually redistributable? 2) is the conversation about upstream involvement public?

Reply:
It is, there's no public conversation about it, but @LukePulverenti (upstream) can attest to it.